cmd/main.go

```diff
@@ -20,6 +20,8 @@ var (
 	outOfCluster   bool
 	version        string
 	config         spec.ControllerConfig
+	kubeQPS        int
+	kubeBurst      int
```

Comment:
I feel these fields should also be part of the spec.ControllerConfig struct

Reply:
I moved it to CRD. Think that It's not needed to be configured via cli.